VWOSdk/VWO.cs

```diff
@@ -47,15 +49,39 @@ static VWO()
         public static void Configure(ILogWriter logger)
         {
             AppContext.Configure(logger);
-        }
 
+            if (_tmpUsageStats != null)
+            {
+                if (logger != null)
+                {
+                    _tmpUsageStats.Add("cl", 1);
+                }
+            }
+            else
+            {
+                _tmpUsageStats = new Dictionary<string, int>();
+                _tmpUsageStats.Add("cl", 1);
+            }
+        }
         /// <summary>
         /// Configure LogLevel for application.
         /// </summary>
         /// <param name="logLevel">Defines the level of logs. LogLevel can be ERROR, WARNING, INFO and DEBUG.</param>
         public static void Configure(LogLevel logLevel)
         {
             AppContext.Configure(logLevel);
+            if (logLevel.Equals(LogLevel.ERROR))
+            {
+                return;
+            }
+            if (_tmpUsageStats != null)

```

Comment:
can we put this inside a function that accepts a key and set it to 1, this part seems repetitive

Reply:
please check that